pkg/server/user/user_test.go

```diff
@@ -111,28 +111,79 @@ func TestDatabaseOperationsSetPassword(t *testing.T) {
 
 	expectedEmail := "teresa@luizalabs.com"
 	expectedPassword := "secret"
-	if err = createFakeUser(db, expectedEmail, "123456"); err != nil {
+
+	if err = createFakeUser(db, "Test", expectedEmail, "123456", false); err != nil {
 		t.Fatal("error on create fake user: ", err)
 	}
 
-	if err = dbu.SetPassword(expectedEmail, expectedPassword); err != nil {
+	user := &database.User{Email: expectedEmail}
+	if err = dbu.SetPassword(user, expectedPassword, ""); err != nil {
 		t.Fatal("error trying to set a new password: ", err)
 	}
 	if _, err = dbu.Login(expectedEmail, expectedPassword, time.Second); err != nil {
 		t.Error("error trying to make login with new password: ", err)
 	}
 }
 
-func TestDatabaseOperationsSetPasswordForInvalidUser(t *testing.T) {
+func TestDatabaseOperationsSetPasswordAdmin(t *testing.T) {
```

Comment:
This Admin suffix is  a little bit misleading as there as tests with non admin users.

Reply:
Yes, this is a mistake